server/service/osquery_utils/queries.go

```diff
@@ -339,7 +339,7 @@ var detailQueries = map[string]DetailQuery{
 SELECT (blocks_available * 100 / blocks) AS percent_disk_space_available,
        round((blocks_available * blocks_size *10e-10),2) AS gigs_disk_space_available
 FROM mounts WHERE path = '/' LIMIT 1;`,
-		Platforms:  []string{"darwin", "linux", "rhel", "ubuntu", "centos"},
+		Platforms:  append(fleet.HostLinuxOSs, "darwin"),
```

Comment:
Double checking: Was `debian` excluded on purpose from this query?

Reply:
Before (main) and after host refetch on a debian host (using this branch):

<img width="700" alt="Screen Shot 2021-12-07 at 12 47 21" src="https://user-images.githubusercontent.com/2073526/145061205-870cdefb-a940-4e5a-9c29-50dfc3c48126.png">

<img width="700" alt="Screen Shot 2021-12-07 at 12 47 33" src="https://user-images.githubusercontent.com/2073526/145061215-86153a5a-8423-4e69-bedf-d3190ab25803.png">